converters/fromZigbee.js

```diff
@@ -4459,6 +4467,22 @@ const converters = {
             return result;
         },
     },
+    ZNMS12LM_low_battery: {
+        cluster: 'genPowerCfg',
+        type: ['attributeReport', 'readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            const result = {};
+            if (typeof msg.data['batteryAlarmMask'] == 'number') {
```

Comment:
I'm surprised to see the `batteryAlarmMask` is used for reporting a low battery. According to the docs this enable configuration of when alarms are generated, not about the battery state itself:

![image](https://user-images.githubusercontent.com/2892853/92635705-8fefc500-f2d6-11ea-9b27-8269d9dd4657.png)

I expect that the `batteryAlarmState` is used for this: https://github.com/Koenkk/zigbee-herdsman-converters/blob/a2d3c9e1ffaa2a1a61a988f01e5e9c906e5d1572/converters/fromZigbee.js#L495

Can you verify this?

Reply:
There is no data "batteryAlarmState" when it say " low battery, please replace". So please see my other comment.

Reply:
You can rename it from battery_low to battery_alarm, no problem at all, but from the log only:
```
Zigbee2MQTT:debug 2020-09-09 08:21:56: Received Zigbee message from '0x00158d000xxx', type 'attributeReport', cluster 'genPowerCfg', data '{"batteryAlarmMask":1}' from endpoint 1 with groupID 0
Zigbee2MQTT:debug 2020-09-09 08:21:56: No converter available for 'ZNMS12LM' with cluster 'genPowerCfg' and type 'attributeReport' and data '{"batteryAlarmMask":1}'
```
So we know only value 1, i don't replace the battery yet (for debug other value), and it still function normally

Reply:
No problem, thanks for checking!